src/discordtipbot/discordbot.cr

```diff
@@ -533,6 +535,44 @@ class DiscordBot
     end
   end
 
+  def glucky(msg : Discord::Message, cmd_string : String)
+    return reply(msg, "**ERROR**: This command doesn't work in DMs") if private?(msg)
+
+    cmd_usage = "#{@config.prefix}lucky [amount] [people]"
+
+    # cmd[0]: command, cmd[1]: amount, cmd[2]: people"
+    cmd = cmd_string.split(" ")
+
+    return reply(msg, cmd_usage) unless cmd.size > 2
+
+    amount = amount(msg, cmd[1])
+    return reply(msg, "**ERROR**: You have to specify an amount! #{cmd_usage}") unless amount
+
+    people = cmd[2].to_i?
+    return reply(msg, "**ERROR**: The number of people should be a number. obviously.") unless people
+
+    users = active_users(msg)
+
+    amount_each = amount/people
+
+    return lucky(msg, cmd_string) unless users && (users = users.to_a).size >= people
```

Comment:
1. I said you should make it rain :smiley: 
and
2. You should make it reply with a different message if there's simply no one active (no one available to get glucky'd)